template.html

```diff
@@ -1,50 +1,83 @@
-<!doctype html>
-
-<html>
-
+<!DOCTYPE html>
+<html lang="en">
 <head>
-
-	<meta charset="UTF-8">
-
-	<title>${head_title}</title>
-
-	<meta name="description" content="${description}">
-
-	<link rel="canonical" href="https://book.cuberite.org">
-
-	<link rel="stylesheet" href="css/main.css" media="screen">
-	<link rel="stylesheet" href="css/print.css" media="print">
-
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Manual - Cuberite</title>
+	<meta name="description" content="A custom Minecraft compatible game server written in C++. It's a fast, lightweight, and awesome replacement for Vanilla.">
+	<link rel="canonical" href="https://book.cuberite.org/">
+	<link rel="stylesheet" href="https://cuberite.org/css/style.css">
+	<link rel="stylesheet" href="https://cuberite.org/css/light.css">
+	<link rel="alternate" type="application/atom+xml" href="https://cuberite.org/feed.xml">
+	<link rel="icon" type="image/png" href="https://cuberite.org/icons/favicon-32x32.png" sizes="32x32">
+	<link rel="icon" type="image/png" href="https://cuberite.org/icons/android-chrome-192x192.png" sizes="192x192">
+	<link rel="icon" type="image/png" href="https://cuberite.org/icons/favicon-96x96.png" sizes="96x96">
+	<link rel="icon" type="image/png" href="https://cuberite.org/icons/favicon-16x16.png" sizes="16x16">
+	<link rel="manifest" href="https://cuberite.org/icons/manifest.json">
+	<link rel="mask-icon" href="https://cuberite.org/icons/safari-pinned-tab.svg">
+	<meta name="theme-color" content="#101010">
 </head>
-
 <body>
-
-	<span style="float: right;">Generated on ${timestamp}</span>
-
-	<h1>${title}</h1>
-
-	<article>
-
-		${content}
-
-	</article>
-
-	<!-- Piwik -->
+	<div id="navigation">
+		<div id="logo">
+			<a href="https://cuberite.org/">Cuberite</a>
+		</div>
+		<div id="navigation-links">
+			<a href="https://forum.cuberite.org/" class="forum">Forum</a>
+			<a href="https://book.cuberite.org/" class="manual active">Manual</a>
+			<a href="https://cuberite.org/support/" class="support">Support</a>
+			<a href="https://cuberite.org/plugins/" class="plugins">Plugins</a>
+			<a href="https://cuberite.org/contribute/" class="contribute">Contribute</a>
+		</div>
+	</div>
+	<div id="manual">
+		<div id="manual-body">
+			<h2 class="manual">${title}</h2>
+			${content}
+			<div id="main-footer">Generated on ${timestamp}</div>
```

Comment:
I think the date is a rather important part of the manual, it would be better at the top rather than after the content.